src/types.d.ts

```diff
@@ -103,18 +103,22 @@ type StatsEndpointResp<Data> =
     }
   | { timeLeft: string };
 
-type DashboardBlockListEntry = {
-  /** mailia.bsky.social */
-  Handle: string;
-  /** https://bsky.app/profile/did:plc:i3bauhmsixt5j33pnr5g7475 */
-  ProfileURL: string;
-  /** 1589 */
-  block_count: number;
-  /** did:plc:i3bauhmsixt5j33pnr5g7475 */
+interface BlockData {
+  count: number;
+  handle: string;
```

Comment:
as @thieflord06 mentions, remove this line as we intend to remove the field from the returned api data soon